### DISCUSSION TITLE
How to get object local coordinate?

### DISCUSSION THREAD

**Discussion by tjzzhang** [bnpr/Malt#422] [Q&A]
The **object postion** in the malt node is the world coordiante, is it possible transform it to local coordinate?

**Comment by pragma37** [comment]
If you are using the Release you can do it like this:
https://github.com/bnpr/Malt/discussions/159#discussioncomment-1665679

In the Development version with new API it's much nicer:
![imagen](https://user-images.githubusercontent.com/36610999/196540738-270f774d-abeb-4232-8ac9-3df11a9f36dd.png)
